std/math.d

```diff
@@ -8751,97 +8756,6 @@ bool approxEqual(T, U, V)(T value, U reference, V maxRelDiff = 1e-2, V maxAbsDif
     }
 }
 
-///
-@safe pure nothrow unittest
```

Comment:
Mark the unittests as deprecated instead of removing them

Reply:
Ah, better! I'll do that soon, but have to leave now.

Reply:
done. But the style checker complains now... Any ideas?

Reply:
I could make the public unittests not public. What is the usual workflow here?

Reply:
I think you can just remove the `///` here.